index.js

```diff
@@ -1,41 +1,74 @@
-const Books = [];
-function Library(books) {
-  this.books = books;
-  this.display = function (arg = null) {
-    if (arg === null) {
-      for (let i = 0; i < Books.length; i++) {
+class Library {
+  books;
+
+  constructor() {
+    this.books = [];
+  }
+
+  display(initialObj, arg) {
+    if (arg === null && initialObj !== null) {
+      for (let i = 0; i < initialObj.length; i += 1) {
         document.getElementById('books').innerHTML += `
-					<h3>${this.books[i].author}</h3>
-					<p>${this.books[i].title}</p>
-					<button id='remove' onClick='bi.remove(${i})'>Remove</button>
-					<hr>
-					`;
+            <li class = "bck">
+              <div class ="main">
+              <p>${initialObj[i].author}</p>
+              <p>by ${initialObj[i].title}</p>
+              </div>
+              <button id='remove' onClick='bi.remove(${i})'>Remove</button>
+            </li>
+            `;
+      }
+    } else if (arg === null) {
+      localStorage.setItem('datas', JSON.stringify(this.books));
+      document.getElementById('books').innerHTML = '';
+      for (let i = 0; i < JSON.parse(localStorage.getItem('datas')).length; i += 1) {
+        document.getElementById('books').innerHTML += `
+            <li class = "bck">
+              <div class ="main">
+              <p>${JSON.parse(localStorage.getItem('datas'))[i].author}</p>
+              <p> by ${JSON.parse(localStorage.getItem('datas'))[i].title}</p>
+              </div>
+              <button id='remove' onClick='bi.remove(${i})'>Remove</button>
+            </li>
+            `;
       }
     } else {
+      localStorage.setItem('datas', JSON.stringify(this.books));
       document.getElementById('books').innerHTML = '';
       for (let i = 0; i < arg.length; i += 1) {
         document.getElementById('books').innerHTML += `
-						<h3>${this.books[i].author}</h3>
-						<p>${this.books[i].title}</p>
-						<button id='remove' onClick='bi.remove(${i})'>Remove</button>
-						<hr>
-						`;
+        <li class = "bck">
+          <div class ="main">
+            <p>${arg[i].author}</p>
+            <p>by ${arg[i].title}</p>
+            </div>
+            <button id='remove' onClick='bi.remove(${i})'>Remove</button>
+          </li>
+            `;
       }
     }
-  };
+  }
 
-  this.remove = function (ref) {
+  remove(ref) {
+    this.books = JSON.parse(localStorage.getItem('datas'));
     this.books.splice(ref, 1);
```

Comment:
# Optional:  Please use `slice()` method because it doesn't mutate the array
- `slice` returns a piece of the array but it doesn't affect the original array. `splice` changes the original array by removing, replacing, or adding values and returns the affected values.
- [Read more Here](https://www.educative.io/answers/splice-vs-slice-in-javascript#:~:text=slice%20returns%20a%20piece%20of,one%20is%20up%20to%20you.)